### PR TITLE
Fix the Jenkins Cobertura Plugin error:Source code is unavailable [Updated PR#8 to check error returns]

### PR DESCRIPTION
**Pull request by abhijithda**
Updated to check for error returns based on review comments in #8 (which had rebased PR #4)

### CODE REVIEW

**Comment by abhijithda** [comment]
Added error checks...

File: gocov-xml.go [hunk 1]
```diff
@@ -60,15 +66,34 @@ type Method struct {
...
 }
 
 func main() {
+	// check the arguments.
+	bpath, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	if len(os.Args) > 2 {
+		if os.Args[1] == "-b" {
```
Comment (AlekSi) [inline gocov-xml.go]: Is there a reason not to use [`flag` package](https://golang.org/pkg/flag/)?

Comment (abhijithda) [reply gocov-xml.go]: 👍 
Had merged from different branch.
I've now updated to use flag package.

File: README.md [hunk 2]
```diff
@@ -59,4 +59,3 @@ gocov test github.com/gorilla/mux | gocov-xml -source /abs/path/to/source > cove
 - [Marin Bek (marinbek)](https://github.com/marinbek)
 - [Alex Castle (acastle)](https://github.com/acastle)
 - [Billy Yao (yaoyaozong)](https://github.com/yaoyaozong)
-- [Abhijith DA (abhijithda)](https://github.com/abhijithda)
```
Comment (AlekSi) [inline README.md]: That's totally fine to keep yourself there. :)

**Comment by AlekSi** [comment]
Thank you!

File: README.md (hunk 2 above)
Comment (abhijithda) [reply README.md]: 👍 
Thanks.

**Pull request merged**